client/src/components/Home/index.tsx

```diff
@@ -213,6 +257,44 @@ export default class Home extends React.Component<any, IHomeState> {
     return (
       <div>
         <h1>Bot Analytics</h1>
+        
+      <Button
```

Comment:
UX suggestion: Atm the import button is placed inside the content page which gets placed within a section of templates, which doesn't seem like the right place. Rather the button could be placed in a sub-toolbar (like the dashboard) and aligned to the right (to keep it in the same region as the export button).